.github/workflows/container_scanning.yaml

```diff
@@ -0,0 +1,39 @@
+name: Docker container scanning
+
+on:
+  pull_request:
+  push:
+    branches: [main, radix]
+  workflow_dispatch:
+
+jobs:
+  vulnerability_scanning:
+    name: Trivy test
+    runs-on: ubuntu-latest
+
```

Comment:
We don't need this `security-events` permission because we don't plan to upload anything to Security tab because we want to fail this job, and not just report to security?
https://github.com/equinor/appsec-ghas-examples/blob/4264a4e047e09d17986758c58eb2b422ffa29562/.github/workflows/trivy-config.yml#L38

If so, :+1: